django/db/models/expressions.py

```diff
@@ -995,14 +995,15 @@ def set_source_expressions(self, exprs):
     def get_source_expressions(self):
         return [self.expression]
 
-    def as_sql(self, compiler, connection):
+    def as_sql(self, compiler, connection, **kwargs):
```

Comment:
What's the value in accepting `**kwargs` as opposed the the specific values that are used (`template`)?